js/arena.js

```diff
@@ -0,0 +1,60 @@
+ import Gladiator from './gladiator.js';
+
+
+export  class Arena {
+   constructor(name ){
+        this.name=name.toUpperCase();
```

Comment:
this will make the entire name of the Arena Uppercase, we want only the first letter.

a way you can do this
```
this._name.charAt(0).toUpperCase() + this._name.slice(1)
```